schema/sample/safety.md

```diff
@@ -0,0 +1,11 @@
+ghs (object):
+
+- pictogram (array<object>)
+  - code (string)
+  - description (string)
+- hStatement (array<object>)
```

Comment:
if you use the plural in pubchem i would also use it here: 

`hStatements (array<object>)`